src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -22,17 +23,17 @@ public partial struct Guid : IFormattable, IComparable, IComparable<Guid>, IEqua
         ////////////////////////////////////////////////////////////////////////////////
         //  Member variables
         ////////////////////////////////////////////////////////////////////////////////
-        private int _a; // Do not rename (binary serialization)
```

Comment:
Does this change affect binary serialization?

Reply:
Ugh, it does.  I didn't think it did, but alas.  Will fix...